openmoc/openmoc.i

```diff
@@ -19,6 +19,7 @@
   #include "../src/LocalCoords.h"
   #include "../src/log.h"
   #include "../src/Material.h"
+  #include "../src/MOCKernel.h"
```

Comment:
Do we need to wrap these new files with SWIG? Or should these algorithms all be under-the-hood and opaque to Python?